docs/changelog.rst

```diff
@@ -4,11 +4,14 @@
 Changelog
 =========
 
+- :release:`9.5.1 <2nd March 2023>`
+- :bug:`174` Bump Discord.py to :literal-url:`2.2.2 <https://github.com/Rapptz/discord.py/releases/tag/v2.2.2>`.
+
 - :release:`9.5.0 <28th February 2023>`
 - :feature:`173` Bump Discord.py to :literal-url:`2.2.0 <https://github.com/Rapptz/discord.py/releases/tag/v2.2.0>`.
 
 - :release:`9.4.1 <9th February 2023>`
-- :feature:`172` Bump Discord.py to :literal-url:`2.1.1 <https://github.com/Rapptz/discord.py/releases/tag/v2.1.1>`.
+- :bug:`172` Bump Discord.py to :literal-url:`2.1.1 <https://github.com/Rapptz/discord.py/releases/tag/v2.1.1>`.
```

Comment:
Was this the gif thing ?

Reply:
Yea. Also, since it was marked as a feature it was getting rolled into 9.5.0, since releases doesn't see a feature as a valid patch release. See https://bot-core.pythondiscord.com/main/changelog.html

Reply:
Oh I see